tools/setup/postgres-init-dev-db

```diff
@@ -39,6 +39,12 @@ fi
 
 DBNAME_BASE=${DBNAME}_base
 
+CHECK_DBRUN=$(pg_isready -U "$DEFAULT_USER" -q | cat; echo "${PIPESTATUS[0]}")
+if [[ $CHECK_DBRUN != 0 ]]; then
+  echo "PostgreSQL Server not running! Ensure the service is enabled."
+  exit
+fi
```

Comment:
You can write this check more simply using `if pg_isready -U $DEFAULT_USER -q` -- no need for the `cat` and `PIPESTATUS`.  I'll just clean this up when merging.